docs/glossary.md

```diff
@@ -24,3 +24,27 @@ manageability created need for BMC (Board Management
 Controllers), EC (Environmental Controllers) and even more
 specialized one like USB Power Delivery firmware. What may
 make that firmware also covered by above definition.
+
+## Dasharo Hardware Compatibility List Report
+
+Dasharo HCL Report dumps most important information about platform and backup
+SPI NOR flash. Gathered information can be used for future analysis, debugging
+and recovery. Optionally scripts upload dump to [Dasharo HCL Backup
+Server](https://backup.dasharo.com), so Dasharo Team can improve open source
```

Comment:
@pietrushnic Does backup.daharo.com exists? I cannot open it. For now we gather reports on nextcloud

Reply:
No, it does not. I explained it here: https://github.com/Dasharo/docs/pull/310/commits/df8b632673ef1155cc53297d0faa585c588683fd